libraries/AC_AttitudeControl/AC_PosControl.h

```diff
@@ -48,7 +47,7 @@ class AC_PosControl
 public:
 
     /// Constructor
-    AC_PosControl(const AP_AHRS_View& ahrs, const AP_InertialNav& inav,
+    AC_PosControl(AP_AHRS_NavEKF& ahrs, const AP_AHRS_View& ahrs_view,
```

Comment:
ok I see, on case on quadplane use a rotation ... damned ! it would have simplified a lot !